src/audio/commands/Music/importqueue.ts

```diff
@@ -1,36 +1,36 @@
-import { MusicCommand, QueueEntry } from '#lib/audio';
+import { AudioCommand, QueueEntry } from '#lib/audio';
 import { GuildSettings, readSettings } from '#lib/database';
 import { LanguageKeys } from '#lib/i18n/languageKeys';
 import type { GuildMessage } from '#lib/types/Discord';
 import { Events } from '#lib/types/Enums';
-import { empty, filter, map, take } from '#utils/common';
-import { isDJ } from '#utils/functions';
+import { empty, filter, map, seconds, take } from '#utils/common';
+import { getAudio, isDJ } from '#utils/functions';
 import { ApplyOptions } from '@sapphire/decorators';
 import { fetch, FetchResultTypes } from '@sapphire/fetch';
 import type { Track } from '@skyra/audio';
 import { deserialize } from 'binarytf';
 import { maximumExportQueueSize } from './exportqueue';
 
-@ApplyOptions<MusicCommand.Options>({
+@ApplyOptions<AudioCommand.Options>({
 	aliases: ['iq'],
-	cooldown: 30,
+	cooldownDelay: seconds(30),
```

Comment:
Is this correct? Is it ms now?

Reply:
Yes, this was changed in https://github.com/sapphiredev/utilities/pull/130

the code: https://github.com/sapphiredev/utilities/blob/ca2a1d6f34864a729526c82f926a959fdacff4f9/packages/ratelimits/src/lib/RateLimitManager.ts#L5-L8